crates/biome_analyze/CONTRIBUTING.md

```diff
@@ -161,16 +161,16 @@ We would like to set the options in the `biome.json` configuration file:
 }
 ```
 
-The first step is to create the Rust data representation of the rule's options.
+The first step is to create the Rust data representation of the rule's options. Each option must be wrapped in a `Option`, this is required so the configuration schema won't mark them as required.
```

Comment:
And then I see this as motivation for the `Option` wrappers :) Could we make the configuration schema respect the `default` annotations instead?